Fundamentals/Data/Defining-content/index.md

```diff
@@ -208,15 +214,30 @@ and this time we'll add two properties. First, make a property called Summary us
 
 ### Creating child nodes
 
-Before we can create a Text Page in the __Content__ section, we need to allow the Text Page Document Type to be created as a child node to the Home node. Select the Home Document Type and go to the __Permissions__ group. Click __Add child__ and select Text Page.
+Before creating a Text Page in __Content__ section, allow the Text Page Document Type to be created as a child node to the Home node. Select the Home Document Type and go to the __Permissions__ group. Click __Add child__ and select Text Page.
 
 ![Allowing child nodes](images/v8Screenshots/setPagePermissions.png)
 
-Go to the __Content__ section and click the menu icon (•••) next to the *Home* node and select the Text page Document Type. We'll name the page "About us". We now have a very basic content structure.
+Go to the __Content__ section and click the menu icon (•••) next to the *Home* node and select the Text page Document Type. We'll name the page "About us". We now have a basic content structure.
 
 ![Basic content structure](images/v8Screenshots/createAboutUs.png)
 
-Document Types are very flexible and can be used in a myriad of ways from defining a piece of reusable content or an entire page, to acting as a container or repository.
+Document Types are flexible and can be used for defining pieces of reusable content or an entire page, to acting as a container or repository.
+
+## What is an Element Type?
+
+An Element Type is a Document Type without a template containing smaller schema configurations for repeating a set of properties. These can be used for defining schema in the Block List Editor, Nested Content, Block Grid Editor, or other Element Type based editors. Element Types cannot be used to create content that resides in the Content tree.
+When you create an Element type, it automatically sets the **Is Element Typw** flag to **True** on the **Permissions** tab.
+
+![Element Type](images/Element-Type.png)
+
+## Creating an Element Type
+
+An Element Type is created under Document Types in the __Settings__ section.
+
+Go to the __Settings__ section in the backoffice. On the __Document Types__ node click the menu icon (•••) to bring up the context menu. Here, choose __Element Type__. This will create an Element Type.
+
+![Creating an Element Type](images/Element-Type-create.png)
```

Comment:
I don't think it's necessary to add steps on how to create an Element Type.
The entire article is about how to create a Document Type, which is pretty much the same workflow.

Perhaps instead, something like this could be added:
"Element Types are created using the same workflow as regular Document Types, but usually contain fewer properties. You can also create Element Types as part of configuring a Block Grid or Block List Data Type."

What do you think? 🤔

Reply:
Yup, makes sense. I'll remove the section and add the suggested line instead 💪